.gitlab-ci.yml

```
@@ -377,7 +377,7 @@ test:acceptance:golang:
 
 test:acceptance:cpp:
   extends: .test:acceptance
-  image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/debian:bullseye
+  image: arm32v7/debian:bullseye
```

Shouldn't we keep the dependency proxy?
```suggestion
  image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/arm32v7/debian:bullseye
```

https://docs.gitlab.com/user/packages/dependency_proxy/